Source/CarthageKit/GitHub.swift

```diff
@@ -53,14 +53,13 @@ extension Repository {
 		}
 
 		// GitHub Enterprise
-		breakpoint: if let url = URL(string: identifier), let host = url.host {
-			var pathComponents = url.pathComponents.filter { $0 != "/" }
-			guard pathComponents.count >= 2 else { break breakpoint }
-
-			// Consider that the instance might be in subdirectories.
-			let name = pathComponents.removeLast()
-			let owner = pathComponents.removeLast()
-
+		if
```

Comment:
@andersio done in https://github.com/Carthage/Carthage/pull/2061/commits/2426c491b73e54b5e7dc7e40312986b47e165ec2